HomeWork 5/LoggerAplication/LoggerAplication/SaveLog.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace LoggerAplication
+{
+    internal class SaveLog
+    {
+        private static bool _checkOutRequest = true;
+
+        public static void SaveLogToFile()
```

Comment:
Actually we are not only saving file here but also checking the necessity to do so

Reply:
separet two operetions! complited